squad/api/rest.py

```diff
@@ -102,7 +102,8 @@ class Meta:
         fields = {'job_id': ['exact', 'in', 'startswith'],
                   'job_status': ['exact', 'in', 'startswith'],
                   'data_processed': ['exact', 'in'],
-                  'status_recorded': ['exact', 'in']}
+                  'status_recorded': ['exact', 'in'],
+                  'completed': ['exact', 'in']}
```

Comment:
"in" makes little sense in this case as 'completed' can only have 2 values.

Reply:
@mwasilew I thought exactly the same when adding this but i followed the pattern of the two booleans above it: 'data_processed' and 'status_processed'. Should I fix all  those three to be only 'exact' as well?

Reply:
I guess so. For Booleanfield there should only be 'exact'.